### PR TITLE
curvefs/tool: fix copyset health check error

### LINKED ISSUE #1022: curvefs/tool: copyset health check fail
**Describe the bug (描述bug)**

image

**To Reproduce (复现方法)**

**Expected behavior (期望行为)**

**Versions (各种版本)**
OS:
Compiler:
curve-mds:
curve-chunkserver:
curve-snapshotcloneserver:
curve-sdk:
nebd:
curve-nbd:

**Additional context/screenshots (更多上下文/截图)**

### PR DESCRIPTION
**Pull request by Cyber-SiKu**
### What problem does this PR solve?

Issue Number: close #1022 #1026 

Problem Summary:
when get all coysetinfo (in curvefs_copyset_status) has error,
tools would not to check copyset status,
so get health instaed of unhealthy

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by Cyber-SiKu** [comment]
recheck

File: curvefs/src/tools/status/curvefs_copyset_status.cpp [hunk 1]
```diff
@@ -41,46 +41,47 @@ void CopysetStatusTool::PrintHelp() {
...
-            if (copyset::checkCopysetHelthy(i.second, key2Status[i.first]) !=
-                copyset::CheckResult::kHealthy) {
-                isHealth = false;
-                break;
-            }
+    bool isHealth = true;
+    for (auto const& i : key2Info) {
+        if (copyset::checkCopysetHelthy(i.second, key2Status[i.first]) !=
+            copyset::CheckResult::kHealthy) {
+            isHealth = false;
```
Comment (SeanHai) [inline curvefs/src/tools/status/curvefs_copyset_status.cpp:60]: The unhealthy copysets can be recorded and show the detail if needed.

File: curvefs/src/tools/status/curvefs_copyset_status.cpp [hunk 2]
```diff
@@ -41,46 +41,47 @@ void CopysetStatusTool::PrintHelp() {
...
 
 int CopysetStatusTool::RunCommand() {
-    if (copyInfoListTool_->RunCommand() == 0) {
-        auto response = copyInfoListTool_->GetResponse();
-        std::map<
-            uint64_t,
-            std::vector<curvefs::metaserver::copyset::CopysetStatusResponse>>
-            key2Status;
-        copyset::CopysetInfo2CopysetStatus(*response.get(), &key2Status);
+    copyInfoListTool_->RunCommand();
```
Comment (wu-hanqing) [inline curvefs/src/tools/status/curvefs_copyset_status.cpp:44]: why drop the return value?

File: curvefs/src/tools/status/curvefs_copyset_status.cpp [hunk 3]
```diff
@@ -41,46 +41,47 @@ void CopysetStatusTool::PrintHelp() {
...
+    auto response = copyInfoListTool_->GetResponse();
+    std::map<uint64_t,
+             std::vector<curvefs::metaserver::copyset::CopysetStatusResponse>>
+        key2Status;
+    copyset::CopysetInfo2CopysetStatus(*response.get(), &key2Status);
```
Comment (wu-hanqing) [inline curvefs/src/tools/status/curvefs_copyset_status.cpp:49]: ditto, you didn't handle return value

File: curvefs/src/tools/status/curvefs_copyset_status.cpp [hunk 4]
```diff
@@ -41,46 +41,47 @@ void CopysetStatusTool::PrintHelp() {
...
 
-        std::map<uint64_t, std::vector<curvefs::mds::topology::CopysetValue>>
-            key2Info;
+    std::map<uint64_t, std::vector<curvefs::mds::topology::CopysetValue>>
+        key2Info;
 
-        copyset::Response2CopysetInfo(*response.get(), &key2Info);
+    copyset::Response2CopysetInfo(*response.get(), &key2Info);
```
Comment (wu-hanqing) [inline curvefs/src/tools/status/curvefs_copyset_status.cpp:54]: ditto

File: curvefs/src/tools/status/curvefs_copyset_status.cpp (hunk 1 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/status/curvefs_copyset_status.cpp:60]: > The unhealthy copysets can be recorded and show the detail if needed.

now is show all copyset, will add this feature.

File: curvefs/src/tools/status/curvefs_copyset_status.cpp (hunk 2 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/status/curvefs_copyset_status.cpp:44]: > why drop the return value?

fix

File: curvefs/src/tools/status/curvefs_copyset_status.cpp (hunk 3 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/status/curvefs_copyset_status.cpp:49]: fix

File: curvefs/src/tools/status/curvefs_copyset_status.cpp (hunk 4 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/status/curvefs_copyset_status.cpp:54]: > ditto

fix

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Review by SeanHai — APPROVED** [review]

**Review by wu-hanqing — APPROVED** [review]

**Pull request merged**